app/views/pet_applications/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2>Application Details</h2>
+
+<section>
+  <p>Applicant Details:
+  <p><%= @pet_application.name %>
+  <p><%= @pet_application.address %>
+  <p><%= @pet_application.city %>, <%= @pet_application.state %> <%= @pet_application.zip %>
+  <p><%= @pet_application.phone_number %>
+  <p>Applicant Notes:
+  <p><%= @pet_application.description %>
+  <p>Pets Applied For:
+  <% @pet_application.pets.each do |pet| %>
+    <div id="app-<%= @pet_application.id %>-pet-<%= pet.id %>">
```

Comment:
i chose `app-1-pet-1` as the format because i expect there to be multiple instances of `pet-1` with multiple applications on the page in the future. this specifies which app.id

Reply:
this great @elguapogordo!

Reply:
Yeah, really like the naming there

Reply:
Good call on the names... really like that naming scheme